ziti/cmd/create/create_config_controller.go

```diff
@@ -152,6 +159,7 @@ func (options *CreateConfigControllerOptions) addFlags(cmd *cobra.Command) {
 	cmd.Flags().DurationVar(&options.EdgeIdentityEnrollmentDuration, optionEdgeIdentityEnrollmentDuration, edge.DefaultEdgeEnrollmentDuration, "the edge identity enrollment duration, use 0h0m0s format")
 	cmd.Flags().DurationVar(&options.EdgeRouterEnrollmentDuration, optionEdgeRouterEnrollmentDuration, edge.DefaultEdgeEnrollmentDuration, "the edge router enrollment duration, use 0h0m0s format")
 	cmd.Flags().IntVar(&options.MinCluster, optionMinCluster, 0, "minimum cluster size. Enables HA mode if > 0")
+	cmd.Flags().StringVar(&options.Console.Location, "console-location", "DISABLED", "enable and set path to console static files")
```

Comment:
the default should probably be "" here and don't overload a boolean into a path. if the location is set to anything non-empty it's 'set'